### DISCUSSION TITLE
Comfim email direct to

### DISCUSSION THREAD

**Discussion by boogerjosh** [supabase/supabase#16307] [Questions]
Does anyone know why i still direct to _localhost:3000_ ? but not _http://localhost:3000/admin-portal_ instead?
<img width="552" alt="Screenshot 2023-08-08 at 00 18 29" src="https://github.com/supabase/supabase/assets/94213206/26bdf87f-ad44-465c-a178-a3aad842c1cf">

**Comment by GaryAustin1** [comment]
Not sure what client code you are using there.  That is not the correct format for javascript signup redirectTo and not sure what confirmText is.

Also you need to make sure you set the redirect FULL url in the dashboard.
![image](https://github.com/supabase/supabase/assets/54564956/49e78f0e-5b35-486f-b0db-02be5c2c9758)
